spec/requests/users_spec.rb

```diff
@@ -2,7 +2,14 @@
 
 RSpec.describe "/users", type: :request do
   let(:admin) { create(:user, :admin) }
-  let(:valid_attributes) { { email: "john@doe.com", password: "123" } }
+  let(:provider_1) { FactoryBot.create(:provider) }
```

Comment:
Since we are using [this module](https://github.com/rubyforgood/skillrx/blob/76095f54cdda805ece7351a0a2631ffb91d45514/spec/support/factory_bot.rb#L2), you can use short form here: `create(:provider)`